qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -539,7 +540,18 @@ def find_at_pos(self, pos, callback):
 
 class WebEngineTab(browsertab.AbstractTab):
 
-    """A QtWebEngine tab in the browser."""
+    """A QtWebEngine tab in the browser.
+
+    Signals:
+        loadFinishedFake:
+            Used in place of unreliable loadFinished
+        loadProgressFake:
+            Used in place of loadProgress
+    """
+
```

Comment:
Remove this blank like

Reply:
Doing so leads to D204 from flake8.

Reply:
Ah, sorry, my bad - I didn't realize this was a class.